broker-cli/proto/broker.proto

```diff
@@ -889,6 +889,8 @@ message Capacity {
   string outstanding_send_capacity = 10;
   // Total outbound outstanding amount. This is currently in base units (e.g. Satoshis/Litoshis)
   string outstanding_receive_capacity = 11;
+  // true/false for any errors that may have occurred when requesting capacity for a particular currency
+  bool error = 20;
```

Comment:
We dont require an error message (we simply want to know if something happened) but not sure if this is the correct way to go about it

Reply:
I changed my mind on this and added a `status` and `error` prop used for messages. This fits inline with other endpoints